### PR TITLE
WineGe Manager implementation

### PR DESCRIPTION
**Pull request by Nocccer**
Based on [ProtonUp api](https://github.com/AUNaseef/protonup/blob/main/protonup/api.py), this pr implementents the same for wine ge in typescript so far. Later on this can be used to install other tools like proton ge etc.

- [x] added package heroic-wine-downloader
- [x] wrote backend for wine-donwloader which uses heroic-wine-downloader (electron/wine-downloader/utils.ts)
- [x] added prototype frontend to test backend implementation

**Note:** Remove frontend implementation before merging this. Frontend is only there to test backend for now.

-------------------------------------------------------------
Use the following Checklist if you have changed something on the Backend or Frontend:

- [x] Tested the feature and it's working on a current and clean install.
    - not implemented in the product yet.
- [x] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [x] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

File: electron/main.ts [hunk 1]
```diff
@@ -63,6 +63,7 @@ import { listenStdout } from './logger'
 import { logError, logInfo, logWarning } from './logger'
 import Store from 'electron-store'
 import { checkUpdates } from './updater'
+import { fetchReleases } from './winege/api'
```
Comment (Nocccer) [inline electron/main.ts:66]: Make function name unique

Comment (dragonDScript) [reply electron/main.ts:66]: I suggest 'fetchWineReleases'

**Comment by flavioislima** [comment]
@Nocccer 
is this one ready to be reviewed?

**Comment by Nocccer** [comment]
@flavioislima it was before we decided to make a package for the backend. I marked it as wip and come back later if the package is published.

**Comment by Nocccer** [comment]
@flavioislima ready for review. Before we merge i would like to remove the frontend part.
It is better if someone with better frontend knowledge can implement it.

**Comment by Nocccer** [comment]
@flavioislima feel free to movve the frontend. Pr is now up to date with main branch.

**Comment by flavioislima** [comment]
> @flavioislima feel free to movve the frontend. Pr is now up to date with main branch.

Good. I took some time yesterday to work on this branch, I'll update the frontend and will push to it.

**Comment by Nocccer** [comment]
Closed because of newer pr #921

**Pull request closed without merging**